### ISSUE TITLE
Program crashes after save parsing

### ISSUE THREAD

**Issue by PreemWho** [TCA166/CK3-history-extractor#8]
I ran the executable (v2.3.2) with a save file (manual save, no iron-mode), and encountered a crash. I supposedly am in the right path, I directly used it inside the game path.
Game version is 1.13.2, I have Windows 11.

Screenshot:
![image](https://github.com/user-attachments/assets/0d7aaaf7-84bc-4da3-9031-a5ef8d478a74)

Save File & report file:
https://file.io/9bTu0dpaRuwf

**Comment by PreemWho** [comment]
Thanks for your feedback, hope it wont be such a bother. I don't know if I can give you more info but feel free to ask.

**Comment by TCA166** [comment]
Sorry for the delay, I have managed to replicate your problem after some digging. I have failed to notice the fact that you used the french localization, and that was the root of the problem. I have fixed your issue, and your save file does now work, though overall the support for different localization styles is... well lackluster. I shall attempt to improve the situation once I find some free time. Thanks for letting me know about the issue
